tests/test_config_utils.py

```diff
@@ -19,6 +19,15 @@ def clean_config(monkeypatch, request):
     monkeypatch.setattr('bigchaindb.config', original_config)
 
 
+def test_ordered_keys_match_database_config():
+    import bigchaindb
+
+    assert set(bigchaindb._base_database_rethinkdb.keys()) ==\
+        set(bigchaindb._base_database_rethinkdb_keys)
+    assert set(bigchaindb._base_database_mongodb.keys()) ==\
+        set(bigchaindb._base_database_mongodb_keys)
+
+
```

Comment:
Duplicate entries could be shown to the user. In the code below, from the command line module:

```python
        for key in database_keys:
            val = conf['database'][key]
            conf['database'][key] = input_on_stderr('Database {}? (default `{}`): '.format(key, val), val)
```

duplicate elements (keys) will trigger a prompt to the user.

We can check the length:

```python
def test_ordered_keys_match_database_config():
    from bigchaindb import (
        _base_database_rethinkdb, _base_database_rethinkdb_keys,
        _base_database_mongodb, _base_database_mongodb_keys,
    )
    assert len(_base_database_rethinkdb.keys()) ==\
        len(_base_database_rethinkdb_keys)
    assert len(_base_database_mongodb.keys()) ==\
        len(_base_database_mongodb_keys)
    assert set(_base_database_rethinkdb.keys()) ==\
        set(_base_database_rethinkdb_keys)
    assert set(_base_database_mongodb.keys()) ==\
        set(_base_database_mongodb_keys)
```